### ISSUE TITLE
Chestmania bug?

### ISSUE THREAD

**Issue by oband2** [kolton/d2bot-with-kolbot#1667]
Starting in act1, even tho it looks like this:
	Scripts.ChestMania = true; // Open chests in configured areas. See sdk/areas.txt
		Config.ChestMania.Act3 = [79, 80, 81, 92, 93, 84, 85]; // List of act 3 areas to open chests in
		Config.ChestMania.Act4 = []; // List of act 4 areas to open chests in
		Config.ChestMania.Act5 = [115, 116, 119, 125, 126, 127]; // List of act 5 areas to open chests in
		Config.ChestMania.Act1 = [13, 14, 15, 16, 18, 19]; // List of act 1 areas to open chests in
		Config.ChestMania.Act2 = [55, 59, 65, 66, 67, 68, 69, 70, 71, 72]; // List of act 2 areas to open chests in

i removed copies of this script, so its not double in the config.

why it starts act 1?

**Comment by DarkHorseDre** [comment]
prolly 'cos chestmania loops through the arrays sequentially, so changing the order of the config lines doesnt affect execution order.

**Comment by oband2** [comment]
is it possible to change it in the chestmania code? and what would i need to do?

**Comment by DarkHorseDre** [comment]
yeah, change the way chestmania selects the next Config.Chestmania.Actn to what you want
